docs/index.rst

```diff
@@ -79,111 +78,44 @@ Getting Started
   <http://www.plasmapy.org/>`_
 * `Using astropy.units <http://docs.astropy.org/en/stable/units/>`_
 
-.. _toplevel-user-documentation:
-
-******************
-User Documentation
-******************
-
-.. _toplevel-plasma-parameters:
-
-Formulary
----------
-
-The `~plasmapy.formulary` subpackage aims to cover the `NRL Plasma Physics
-Formulary <https://www.nrl.navy.mil/ppd/content/nrl-plasma-formulary>`_.
-
-.. toctree::
-   :maxdepth: 1
-
-   formulary/index
-
-.. _toplevel-experimental-tools:
-
-Experimental Tools
-------------------
-
-The `~plasmapy.diagnostics` package is in the early stages of
-development.
-
 .. toctree::
    :maxdepth: 1
+   :caption: Package features
+   :hidden:
 
    diagnostics/index
-
-.. _toplevel-data-structures:
-
-Data Structures and Simulation
-------------------------------
-
-.. toctree::
-    :maxdepth: 1
-
-    plasma/index
-    simulation/particletracker
-
-.. _toplevel-particle-data:
-
-Particle Data
--------------
-
-.. toctree::
-    :maxdepth: 1
-
-    particles/index
-
-.. _toplevel-utilities:
-
-Utilities
----------
-
-.. toctree::
-   :maxdepth: 1
-
+   formulary/index
+   particles/index
+   simulation/particletracker
+   plasma/index
    utils/index
```

Comment:
I'm really not a fan of the inline code blocks being in the side bar titles.  To me it just make everything look cluttered.  However, I'm okay with them being in the page titles.  Maybe do something like...

```
Diagnostics <diagnostics/index>
Formulary <formulary/index>
Particles <particles/index>
Particle Tracker <simulation/particletracker>
Plasma Objects <plasma/index>
Package Utilities <utils/index>
```

When I do the Langmuir diagnostic PR, I'll probably transition the "Diagnostic" page into an "Analysis & Diagnostics" page since the two are so closely related. (A discussion for another day.)

Reply:
Good idea, hadn't thought of that in this context.